### PR TITLE
docker env fixes

### PR DESCRIPTION
**Pull request by GottZ**
make docker configuration great again

`NPOMF_CONTACTS = ["foo@bar.com", "bar@foo.com"]` etc.

before it was impossible to assign arrays / objects through environment variables.

### CODE REVIEW

**Comment by GottZ** [comment]
@maxpowa I'll create my own fork if you don't react :D

**Comment by RX14** [comment]
@GottZ maxpowa hasn't been on IRC since Friday. I'm sure he'll get to your PR if you're patient.

File: config/core.js [hunk 1]
```diff
@@ -122,6 +114,25 @@ config.SESSION_OPTIONS = {
...
+    try {
+      config[eattr] = JSON.parse(data);
+    } catch (exception) {
+      config[eattr] = data;
+    }
 	}
 }
+
+// lets move it here so changes made to MAX_UPLOAD_SIZE will be displayed correctly.
+if (typeof config.DESCRIPTION == 'undefined') {
```
Comment (maxpowa) [inline config/core.js]: I prefer `=== undefined` rather than `typeof == 'undefined'`, probably best to check null as well.

File: config/core.js [hunk 2]
```diff
@@ -122,6 +114,25 @@ config.SESSION_OPTIONS = {
...
+  config.DESCRIPTION = "Upload whatever you want here, as long as it's under "+
+                       config.MAX_UPLOAD_SIZE/1000000 + "MB.<br/> "+
+                       "Please read our <a href='/faq'>FAQ</a>, as we may "+
+                       "remove files under specific circumstances.";
+}
+
+// DO NOT TOUCH UNLESS YOU KNOW HOW TO PROPERLY CONFIGURE CORS
+// Changes the file upload form to POST to this URL instead of the one it's loaded from.
+if (typeof config.UPLOAD_URL == 'undefined') config.UPLOAD_URL = config.URL;
```
Comment (maxpowa) [inline config/core.js]: ditto

**Review by maxpowa — CHANGES_REQUESTED** [review]

**Review by GottZ — COMMENTED** [review]
i came up with an even better idea. will edit my pull request acordingly. you'll like it.

**Comment by maxpowa** [comment]
LGTM, thanks.

**Comment by GottZ** [comment]
lol I just noticed this would not work. it needs to be negated like if(!(foo in bar))

**Comment by maxpowa** [comment]
Yup that's correct, this is what I get for code reviewing in the morning. I'll fix it.

**Pull request merged**